### ISSUE TITLE
Feature Request - signal-cli-rest-api integration

### ISSUE THREAD

**Issue by agentdr8** [Shaztech/Pilldispenser#16]
Great project you've got going here. All of my 3D printed parts are done, and have received my PCBs. Just waiting on components for the time being. 

Taking a look at the functionality, I see that there is built-in Telegram notifications, using the UniversalTelegramBot library. Would it be possible to include Signal integration, by way of the [signal-cli-rest-api](https://github.com/bbernhard/signal-cli-rest-api)? I have that Docker container running within my network, and would love to be able to fire off notifications via Signal. It could even support per-user or per-tray notifications. 

Since it's just HTTP requests, it shouldn't require any additional libraries, just some code additions. If it's out of scope for what you want to implement, I might be able to submit a PR.

**Comment by Shaztech** [comment]
I can add it, but only I you can send me the code for that portion.
I also have no mean to test it.

Thanks
